packages/manager-react-components/src/components/navigation/menus/changelog/changelog.component.tsx

```diff
@@ -23,9 +24,10 @@ const GO_TO = (link: string) => `go-to-${link}`;
 export const ChangelogButton: React.FC<ChangelogButtonProps> = ({
   links,
   chapters = [],
+  isWithTracking = true,
 }) => {
   const { t } = useTranslation('buttons');
-  const { trackClick } = useOvhTracking();
+  const ovhTracking = isWithTracking ? useOvhTracking() : null;
```

Comment:
I believe this breaks the "don't call hooks conditionally" hook rule https://react.dev/reference/rules/rules-of-hooks